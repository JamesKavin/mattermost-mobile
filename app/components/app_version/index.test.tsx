// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
// See LICENSE.txt for license information.

import React from 'react';

import {renderWithIntl} from '@test/intl-test-helper';

import AppVersion from './index';

describe('@components/app_version', () => {
    it('should match snapshot', () => {
        const wrapper = renderWithIntl(<AppVersion/>);
        expect(wrapper.toJSON()).toMatchSnapshot();
    });
});
