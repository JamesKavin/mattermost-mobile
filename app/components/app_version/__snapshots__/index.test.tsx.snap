// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@components/app_version should match snapshot 1`] = `
<View
  animatedStyle={
    {
      "value": {
        "opacity": 1,
      },
    }
  }
  collapsable={false}
  pointerEvents="none"
  style={
    {
      "opacity": 1,
    }
  }
>
  <View
    style={
      {
        "bottom": 0,
        "marginBottom": 12,
        "marginLeft": 20,
        "position": "absolute",
      }
    }
  >
    <Text
      style={
        {
          "fontSize": 12,
        }
      }
    >
      App Version: 0.0.0 (Build 0)
    </Text>
  </View>
</View>
`;
